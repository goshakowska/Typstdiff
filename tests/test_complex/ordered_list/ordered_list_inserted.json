{"pandoc-api-version":[1,23,1],"meta":{},"blocks":[{"t":"OrderedList","c":[[1,{"t":"DefaultStyle"},{"t":"DefaultDelim"}],[[{"t":"Para","c":[{"t":"Str","c":"The"},{"t":"Space"},{"t":"Str","c":"climate"},{"t":"Space"},{"t":"Str","c":"intensive"}]},{"t":"BulletList","c":[[{"t":"Para","c":[{"t":"Str","c":"Precipitation"},{"t":"Space"},{"t":"Str","c":"important"}]}],[{"t":"Para","c":[{"t":"Str","c":"Temperature"},{"t":"Space"},{"t":"Str","c":"scales"}]},{"t":"OrderedList","c":[[1,{"t":"DefaultStyle"},{"t":"DefaultDelim"}],[[{"t":"Para","c":[{"t":"Str","c":"degree"}]},{"t":"BulletList","c":[[{"t":"Para","c":[{"t":"Str","c":"hot"}]}],[{"t":"Para","c":[{"t":"Str","c":"cold"}]}],[{"t":"Para","c":[{"t":"Str","c":"warm"}]}],[{"t":"Para","c":[{"t":"Str","c":"really"},{"t":"Space"},{"t":"Str","c":"hot"}]}]]}],[{"t":"Para","c":[{"t":"Str","c":"sun"}]}],[{"t":"Para","c":[{"t":"Str","c":"cloud"}]}],[{"t":"Para","c":[{"t":"Str","c":"wind"}]}]]]}]]}],[{"t":"Para","c":[{"t":"Str","c":"The"},{"t":"Space"},{"t":"Str","c":"geology"}]}],[{"t":"Para","c":[{"t":"Str","c":"something"},{"t":"Space"},{"t":"Str","c":"new"}]}]]]}]}
