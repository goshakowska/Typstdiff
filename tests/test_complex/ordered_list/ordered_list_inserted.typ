+ The climate intensive
  - Precipitation important
  - Temperature scales
    + degree 
        - hot
        - cold
        - warm
        - really hot
    + sun
    + cloud
    + wind
+ The geology
+ something new