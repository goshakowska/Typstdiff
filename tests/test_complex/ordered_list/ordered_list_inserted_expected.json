{"pandoc-api-version": [1, 23, 1], "meta": {}, "blocks": [{"t": "OrderedList", "c": [[1, {"t": "DefaultStyle"}, {"t": "DefaultDelim"}], [[{"t": "Para", "c": [{"t": "Str", "c": "The"}, {"t": "Space"}, {"t": "Str", "c": "climate"}, {"t": "Underline", "c": [{"t": "Space"}]}, {"t": "Underline", "c": [{"t": "Str", "c": "intensive"}]}]}, {"t": "BulletList", "c": [[{"t": "Para", "c": [{"t": "Str", "c": "Precipitation"}, {"t": "Underline", "c": [{"t": "Space"}]}, {"t": "Underline", "c": [{"t": "Str", "c": "important"}]}]}], [{"t": "Para", "c": [{"t": "Str", "c": "Temperature"}, {"t": "Underline", "c": [{"t": "Space"}]}, {"t": "Underline", "c": [{"t": "Str", "c": "scales"}]}]}, {"t": "OrderedList", "c": [[1, {"t": "DefaultStyle"}, {"t": "DefaultDelim"}], [[{"t": "Para", "c": [{"t": "Str", "c": "degree"}]}, {"t": "BulletList", "c": [[{"t": "Para", "c": [{"t": "Str", "c": "hot"}]}], [{"t": "Para", "c": [{"t": "Str", "c": "cold"}]}], [{"t": "Para", "c": [{"t": "Str", "c": "warm"}]}], [{"t": "Para", "c": [{"t": "Underline", "c": [{"t": "Str", "c": "really"}]}, {"t": "Underline", "c": [{"t": "Space"}]}, {"t": "Underline", "c": [{"t": "Str", "c": "hot"}]}]}]]}], [{"t": "Para", "c": [{"t": "Str", "c": "sun"}]}], [{"t": "Para", "c": [{"t": "Underline", "c": [{"t": "Str", "c": "cloud"}]}]}], [{"t": "Para", "c": [{"t": "Underline", "c": [{"t": "Str", "c": "wind"}]}]}]]]}]]}], [{"t": "Para", "c": [{"t": "Str", "c": "The"}, {"t": "Space"}, {"t": "Str", "c": "geology"}]}], [{"t": "Para", "c": [{"t": "Underline", "c": [{"t": "Str", "c": "something"}]}, {"t": "Underline", "c": [{"t": "Space"}]}, {"t": "Underline", "c": [{"t": "Str", "c": "new"}]}]}]]]}]}